README.md

```diff
@@ -1,103 +1,55 @@
 # The Charmed Operator Framework
 
-This Charmed Operator Framework simplifies [operator](https://charmhub.io/about) development
-for [model-driven application management](https://juju.is/model-driven-operations).
+The Charmed Operator Framework is a Python library (available as [ops](https://pypi.org/project/ops/))
```

Comment:
```suggestion
The Operator Framework is a Python library (available as [ops](https://pypi.org/project/ops/))
```

Based upon recent changes in the Juju Terminology doc and recent conversations :) applies throughout